README.md

```diff
@@ -48,7 +48,7 @@ Prometheus instances, rather than needing configuration elsewhere.
 
 Configure an alerting rule to always alert if your instance is healthy. The
 simplest is just `expr: 1`. We recommend a slightly more advanced rule (see
-below) while checking the "up" status of yourself may seen redundant it
+below) while checking the "up" status of yourself may seen redundant, it
```

Comment:
This is the current sentence. It's a mouthful.

```
We recommend a slightly more advanced rule (see below) while checking the "up" status of yourself may seen redundant it provides a basic end-to-end sanity check of your Prometheus instance consistency and potentially your service discovery.
```

I feel it may be more appropriate to say

```
We recommend a slightly more advanced rule (see below). While checking the "up" status of yourself may seen redundant, it provides a basic end-to-end sanity check of your Prometheus instance consistency and potentially your service discovery.
```

Reply:
I think putting that full stop in fixes my concern. It prevents the phrase around "up" being linked to the statement around advanced rule.

Reply:
Updated with a full stop